### PR TITLE
fix: context URLs of JetBrains example repos.

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
> Signed-off-by: Siddhant Khare <siddhant@gitpod.io>

## Description

With the recent migration of example/template repositories to `gitpod-samples`. We need to update these URLs. Currently it is working as it is also. because GitHub APIs automatically redirects to the new owner URL. But, eventually text should be updated in UI (new URLs) & would save a extra redirect :smile:

image

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=jetbrains
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by loujaybee — APPROVED** [review]

File: components/ide/jetbrains/image/gha-update-image/index.js [hunk 1]
```diff
@@ -12,56 +12,56 @@ const IDEs = [
...
-        exampleRepo: "https://github.com/gitpod-io/template-jetbrains-rubymine",
+        exampleRepo: "https://github.com/gitpod-samples/template-jetbrains-rubymine",
     },
     {
         productName: "WebStorm",
         productId: "webstorm",
         productCode: "WS",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/template-jetbrains-webstorm",
+        exampleRepo: "https://github.com/gitpod-samples/template-jetbrains-webstorm",
```
Comment (felladrin) [inline components/ide/jetbrains/image/gha-update-image/index.js:50]: These two repositories currently doesn't exist:
- https://github.com/gitpod-samples/template-jetbrains-rubymine
- https://github.com/gitpod-samples/template-jetbrains-webstorm

**Review by felladrin — APPROVED** [review]

File: components/ide/jetbrains/image/gha-update-image/index.js (hunk 1 above)
Comment (Siddhant-K-code) [reply components/ide/jetbrains/image/gha-update-image/index.js:50]: @felladrin I am not sure what we should do. Since, old repos. are also deprecated 😢

Comment (Siddhant-K-code) [reply components/ide/jetbrains/image/gha-update-image/index.js]: Resolved in [`f676b18`](https://github.com/gitpod-io/gitpod/pull/15502/commits/f676b1829246c6cbb20ca7843e956be909ee0624)

Thanks @felladrin for the suggestions ❤️

**Pull request merged**